index.html

```diff
@@ -0,0 +1,108 @@
+            </ul>
+
+        </nav>
+        <nav aria-label="Menu" class="desktop-menu">
+            <a class="logo" href="index.html"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">Campaign</button></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="banner">
+        <video id="background-video" muted loop autoplay playsinline preload="auto" poster="Images/banner_Moment.jpg">
+            <source type="video/mp4" src="Images/banner.mp4" />
+        </video>
+        <div class="banner-page">
+            <i class="menu-button"></i>
+            <div>
+                <h2>Welcome to the</h2>
+                <h1>12th Software Development<br>Summit 2023</h1>
+            </div>
+            <p class="brief-description-banner"> Lorem Ipsum has been the industry's standard dummy text ever since the
+                1500s, when an unknown
+                printer took a galley of type and scrambled it 1960s.</p>
+            <div class="date-location">
+                <h2>2023.04.22(SAT) ~ 23(SUN)</h2>
+                <p>@XPO, Ministry of Inovation & Technology-Ethiopia</p>
+            </div>
+        </div>
+    </section>
+    <section id="main">
+        <h1 class="section-headline">Main Program</h1>
+        <div class="main-container">
+            <div class="program">
+                <i class="i-lecture"></i>
+                <h3>Lecture</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-exhibition"></i>
+                <h3>Exhibition</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-forum"></i>
+                <h3>Forum</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-workshop"></i>
+                <h3>Workshop</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+            <div class="program">
+                <i class="i-ignite"></i>
+                <h3>Lecture</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum reprehenderit.</p>
+            </div>
+        </div>
+        <button type="button">Join ICT Expo 2023</button>
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="featured-speakers">
+        <h1 class="section-headline">Featured Speakers</h1>
```

Comment:
- [ ] Good work on the website so far! However, during my review of the code, I noticed that there are multiple` h1 tags` on each page. For accessibility purposes, it's recommended to have only one` h1 tag `per page, which should be used for the main heading of the page. This helps to ensure that screen readers and other assistive technologies can properly navigate and understand the content. Please consider revising the code to have only one `h1 tag `per page.